### ISSUE TITLE
VPN Tracker 365 won't connect to IPsec gateway while LuLu is enabled

### ISSUE THREAD

**Issue by gmunk** [objective-see/LuLu#271]
Hello, 

So today I've updated Catalina (10.15.7) and LuLu (2.0.0) to the latest available versions. For work I'm oblidged to use a vpn client called VPN Tracker 365. 
Unfortunately when trying to connect to the gateway, while LuLu is enabled and I've allowed all the connections concerned with the VPN client, I'm observing a DHCP timeout which is ilustrated by the following fragment present in the logs of VPN Tracker 365:

```
	17:56:19	Sending DHCP DISCOVER packet, waiting for DHCP OFFER reply
	17:56:19	(Re-)Sending DHCP message...
        17:56:46	Connection timeout hit
```

If I disable LuLu from the status bar and retry the connection, it succeeds. VPN Tracker 365 has this feature called "Connection Checker" which checks whether the current internet connection is capable to connect to various types of gateways. When I run it with LuLu enabled it shows that I won't be able to connect to IPsec gateways since ESP and NAT-T are not supported, if I disable LuLu they turn to being supported.

Now, I know this is a fairly specific issue but I'll be very grateful for any advice since it is annoying to disable my firewall everytime I need to connect to that work VPN. Any help will be greatly appreciated.

**Comment by brettjashford** [comment]
i am also having an issue with cisco ipsec vpn connections, but using the osx network settings to connect instead of a 3rd party app. when connected to the vpn, regular traffic goes through but intranet traffic does not. disabling lulu allows intranet & regular traffic. thanks in advance for any help

**Comment by david-kreindler** [comment]
I am also having the same issue with macOS 11.0.1 and the built-in macOS Cisco IPsec VPN service. The IKE succeeds (I successfully authenticate and set up the VPN), but no traffic is sent into the VPN tunnel. Disabling Lulu is the only thing I have found that works around the problem.

**Comment by ciapamusche** [comment]
I am also having the same issue with macOS 11.0.1 and Ivacy VPN. Disabling Lulu VPN Works correctly.

Any solution?

**Comment by ciapamusche** [comment]
https://www.obdev.at/support/littlesnitch/245917044280819

maybe a bug. someone have the beta installed?

**Comment by gmunk** [comment]
> https://www.obdev.at/support/littlesnitch/245917044280819
> 
> maybe a bug. someone have the beta installed?

I'm on Catalina, but I guess LuLu was updated to ditch the kext-based implementation it was using. In any case I started noticing this behaviour after updating to 2.0.0. Maybe someone with Big Sur can weigh in.

**Comment by ciapamusche** [comment]
In Big Sur Not working. But seems I have found Walkaround in my case.
The issue seems to be appear with the IKeV2 protocol.
Manually add a L2TP VPN connection with your VPN provider.  I have Ivacy and this solve the problem at the moment.

**Comment by greatwaldini** [comment]
I logged onto GH to report possibly a similar issue.  I'm on Catalina 10.15.7, LuLu 2.0.0 and in the last few days I've noticed my machine hanging up while web browsing.  A little bit of roughshod trouble shooting led me to realize this was not happening with LuLu disabled.  Of note, I recently switched from NordVPN (no issues with LuLu in more or less standard configurations of UDP/TCP and OpenVPN) to Mullvad VPN which runs WireGuard.  I'm not entirely sure if this is of significance but it seems it may be.. Mullvad doesn't seem to give any sort of debug logs, and I'm not seeing anything out of the ordinary via network monitoring.  I've ensured that all rules for all apps are set with proper permissions in LuLu, and I'm out of ideas - hence coming here.   Hopefully this helps.

**Comment by ciapamusche** [comment]
With last Bigsur update the issue is not present... 

Ivacy VPN and Lulu seems works.

**Comment by silv3r23** [comment]
> With last (11.1) Bigsur update the issue is not present...

confirm IPsec works now while having LuLu enabled

**Comment by objective-see** [comment]
Yes, this was an Apple bug 😢 
...but as noted, it has been fixed in macOS 11.1/11.2!
